components/generator/Select.vue

```diff
@@ -12,7 +12,7 @@
       :value="value"
       @input="$emit('input', $event.target.value)"
     >
-      <option v-for="item of items" :key="item" :value="item" v-text="item" />
+      <option  class="bg-gray-800" v-for="item of items" :key="item" :value="item" v-text="item" />
```

Comment:
Can this not be set on the `select`?

Reply:
I just figured out there is a problem with `bg-gray-800/10` in `select` it didn't apply on `option` I don't know why but we can just change it to `bg-gray-900` it's the same color as `bg-gray-800/10`.